openslides_backend/services/datastore/adapter.py

```diff
@@ -379,7 +386,8 @@ def update_locked_fields_from_mapped_fields(
             for field in mapped_fields:
                 if not field.startswith("meta_"):
                     self.update_locked_fields(
-                        FullQualifiedField(fqid.collection, fqid.id, field), position
+                        to_fqfield(fqid_collection(fqid), fqid_id(fqid), field),
```

Comment:
add another helper method analogous to https://github.com/OpenSlides/openslides-datastore-service/blob/ddd9b10c69dd98e3acf9c4fb7d024a87cad3aff0/datastore/shared/util/key_transforms.py#L10

Reply:
done